app/views/students/show.html.erb

```diff
@@ -1,34 +1,83 @@
-<div class="container">
-  <div class="d-flex my-5">
-    <div class="mr-5">
-      <% if current_user.profile_photo? %>
-        <%= cl_image_tag current_user.photo.key, height: 150, width: 150, crop: :thumb, gravity: :face, class: "avatar-xl" %>
-      <% else %>
-        <i class="fas fa-user-circle" style="font-size: 150px;"></i>
-      <% end %>
-    </div>
-    <div class="d-flex flex-column justify-content-center">
-      <h5><%= @student.first_name %> <%= @student.last_name %></h5>
-      <p><%= current_user.email %></p>
-      <%= link_to 'Edit Profile', edit_user_registration_path, class: "btn btn-outline-dark" %>
-    </div>
-  </div>
-  <div>
-    <h6><%= pluralize @student.groups.size, 'Class' %></h6>
-    <% @student.groups.each do |group| %>
-      <p>
-        <%= group.class_name %>
-        <%= link_to student_students_group_path(@student, @students_groups.find_by(group: group)),
-            method: :delete,
-            data: { confirm: "Are you sure you want to leave this class?" } do %>
-            <i class="fas fa-sign-out-alt"></i>
+
+
+
+<div class="container mt-4">
```

Comment:
I'd keep mt-5 out of standardization - it's like that in almost every page (execpt when there's a back button on top, cos that kinda creates a margin for himself). Of course that this can be changed, but then it should be changed everywhere, so for now I'd keep 5.

Reply:
ok good point